src/cljs/coffeepot/views/logged-user.cljs

```diff
@@ -0,0 +1,50 @@
+(ns coffeepot.views.logged-user
```

Comment:
This namespace is problematic. We already have coffeepot.views (The main views.cljs file). Maybe this folder could be renamed or the main views.cljs could be repurposed as a sort of a main container view and renamed.